webapps/frontend/app/views/site/backend/backend-alba-detail-registered-node.html

```diff
@@ -162,6 +162,38 @@
                                     </a>
                                 </li>
                             <!-- /ko -->
+                            <!-- ko if: ![undefined, null].contains(device()) -->
+                                <li>
+                                    <a>
+                                        <span data-i18n="alba:generic.device"></span>:
+                                        <span data-bind="shortText: device(), maxLength: 23, middle: true"></span>
+                                    </a>
+                                </li>
+                            <!-- /ko -->
+                            <!-- ko if: ![undefined, null].contains(mountpoint()) -->
+                                <li>
+                                    <a>
+                                        <span data-i18n="alba:generic.mountpoint"></span>:
+                                        <span data-bind="shortText: mountpoint(), maxLength: 23, middle: true"></span>
+                                    </a>
+                                </li>
+                            <!-- /ko -->
+                            <!-- ko if: status() !== 'empty' && usage() !== undefined -->
+                                <li>
+                                    <a>
+                                        <span data-i18n="ovs:generic.usage"></span>: <span data-bind="text: $root.formatBytes(usage().used)"></span> / <span data-bind="text: $root.formatBytes(usage().size)"></span>
+                                        - <span data-bind="text: $root.formatPercentage(usage().used / usage().size)"></span>
+                                    </a>
+                                </li>
+                            <!-- /ko -->
+                            <!-- ko if: ![undefined, null].contains(slotID()) -->
+                                <li>
+                                    <a>
+                                        <span data-i18n="alba:generic.id"></span>:
```

Comment:
I'd put the slotID on top, no?

Reply:
There are buttons bound to the status. These buttons are most clear in the top right corner. But it can be on the third line.